src/App.js

```diff
@@ -3,8 +3,22 @@ import { Routes, Route } from "react-router-dom"
 import LeaderBoard from './Pages/LeaderBoard';
 import ChosenProtocol from './Pages/ChosenProtocol';
 import TotalVAR from './Pages/TotalVAR';
+import { createClient } from "urql"
+import { useNavigate } from "react-router-dom"
+import React, { useState } from 'react'
+
+const APIURL = "https://api.thegraph.com/subgraphs/name/greg-torrington/greg-v3"
+
+const client = createClient({
+  url: APIURL
+})
```

Comment:
This should work fine with the way you are doing it but checkout the urql docs. You should be able to setup a provider that wraps around your application and then making queries can be simplified using the libraries hooks.